deployments/overlays/base/README.md

```diff
@@ -0,0 +1,4 @@
+# Nodeport Kustomization
```

Comment:
why do we need the folder `deployments/overlays/base`?  we already have `deployments/base`

If you wanna have some common labels. I think you can use `commonLabels`

Reply:
I have this folder here to apply all the same kustomizations we have in the other two overlays, and make the usage more consistent. I'm actually don't have namespace listed in either of the configurations, so applying from base will put it in default.

Reply:
>I have this folder here to apply all the same kustomizations we have in the other two overlays, and make the usage more consistent.

I think this is exactly the use case for deployment/base folder. 

I don't see a reason why you can't define namespace in its own respective folder.  User either choose ingress or nodeport, you can't apply both. I don't think there is any user needed to have a place to change and apply both ingress and nodeport. Especially in this folder/file here, only common label/annotation/namespace stuff, I don't think there is any use cases if you define a service in namespace defined by base, and user want to apply ingress in a different namespace.